generic_camera_model_sampler.py

```diff
@@ -0,0 +1,67 @@
+
+from colorama import Fore, Style
+
+from .planar_as_base import ( PlanarAsBase, INTER_MAP, input_2_torch, torch_2_output )
+from .camera_model_sampler import CameraModelRotation
+from .register import (SAMPLERS, register)
+from ..mvs_utils.ftensor import FTensor
+
+@register(SAMPLERS)
+class GenericCameraModelSampler(CameraModelRotation):
+    def __init__(self, camera_model_raw, camera_model_target, R_output_in_input, preprocessing = lambda x: x, postprocessing = lambda x: x):
```

Comment:
PyTorch doesn't like lambda objects. Especially when this piece of code is used in a dataset implementation working in parallel. The reason is PyTorch does not know how to serialize the lambda function during parallel/inter-process communication.

Reply:
Oh, okay. I learned a new thing.
Would the better way to go then, be to set the args to `None` and replace those with a member identity function if those are not populated? Maybe only call those if they are passed? Or maybe to declare a method outside the class that is just `def identity(x): return x`?

Maybe something else completely?

Reply:
def identity(x): return x is good enough 😀